decidim-assemblies/app/views/layouts/decidim/_assembly_header.html.erb

```diff
@@ -39,7 +39,7 @@
         <div class="row column process-nav__content is-active" id="process-nav-content" data-toggler=".is-active">
           <ul>
             <li class="<%= "is-active" if is_active_link?(decidim_assemblies.assembly_path(current_assembly), :exclusive) %>">
-              <%= active_link_to decidim_assemblies.assembly_path(current_assembly), active: :exact, class: "process-nav__link", class_active: "is-active" do %>
+              <%= active_link_to decidim_assemblies.assembly_path(current_assembly), active: :exclusive, class: "process-nav__link", class_active: "is-active" do %>
```

Comment:
Does `Exclusive` mean `unless another element is highlighted`?

Reply:
I think it means url (including optional parameters like `?locale=`) but not "children urls" (like `/processes/my-slug/f/1`).

Reply:
Eactly what @deivid-rodriguez said.

Reply:
But I actually don't really get it. Why it was working for every value except for the default locale?

Reply:
Sincerely? No idea ¯\_(ツ)_/¯

Reply:
:joy: Ok!

Maybe it's good to get covered the code we don't understand... :stuck_out_tongue: I remember adding a `have_selected_option` matcher a while ago to fix a similar issue.

Reply:
I think it's because since the default locale is also the route's default value, the rails' router is not including it when asking for `request.path` even though it's there. Or something like this 🤣

Reply:
Makes sense to me!